src/main/java/com/googleinterns/zoomtube/servlets/MarkAnsweredServlet.java

```diff
@@ -110,4 +91,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     return Optional.empty();
   }
+
+  /**
+   * Updates the question-type comment specified by `commentKey` to `newType`.  Returns
+   * {@Optional.empty()} if there were no errors, otherwise returns an error message.
```

Comment:
Is this supposed to be {@code Optional.empty()}?

Reply:
ah yes, thanks for catching that